### PR TITLE
treefrog 2.6.0

### PR DESCRIPTION
**Pull request by treefrogframework**
Updated to v2.6.0. Link glog shared library.

### CODE REVIEW

File: Formula/treefrog.rb [hunk 1]
```diff
@@ -24,7 +24,11 @@ class Treefrog < Formula
   end
 
   depends_on xcode: :build
+  depends_on "cmake"
```
Comment (SMillerDev) [inline Formula/treefrog.rb:27]: Where is this used?

Comment (chenrui333) [reply Formula/treefrog.rb:27]: needed in the `configure` step

```
==> ./configure --prefix=/opt/homebrew/Cellar/treefrog/2.6.0 --enable-shared-mongoc --enable-shared-glog
cmake: command not found
```

Comment (SMillerDev) [reply Formula/treefrog.rb]: Whyyyyyyyyyy. Okay, but let's add a comment that `# ./configure calls cmake`

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
Thanks @treefrogframework!

**Pull request closed without merging**